common/src/main/java/com/microsoft/identity/common/internal/authorities/AzureActiveDirectoryAuthority.java

```diff
@@ -75,32 +90,18 @@ private void getAzureActiveDirectoryCloud() {
         mAzureActiveDirectoryCloud = cloud;
     }
 
-    public AzureActiveDirectoryAuthority(AzureActiveDirectoryAudience signInAudience) {
-        mAudience = signInAudience;
-        mAuthorityTypeString = "AAD";
-        getAzureActiveDirectoryCloud();
-    }
-
-    public AzureActiveDirectoryAuthority() {
-        //Defaulting to AllAccounts which maps to the "common" tenant
-        mAudience = new AllAccounts();
-        mAuthorityTypeString = "AAD";
-        mMultipleCloudsSupported = false;
-        getAzureActiveDirectoryCloud();
-    }
-
     public Map<String, String> getFlightParameters() {
         return this.mFlightParameters;
     }
 
-    public void setMultipleCloudsSupported(boolean supported) {
-        mMultipleCloudsSupported = supported;
-    }
-
     public boolean getMultipleCloudsSupported() {
         return mMultipleCloudsSupported;
     }
 
+    public void setMultipleCloudsSupported(boolean supported) {
+        mMultipleCloudsSupported = supported;
+    }
+
     @Override
```

Comment:
Any idea what different purposes these two methods solve getAuthorityUri vs getAuthorityURL?
I am thinking we can just use one.

Reply:
We might like to shrink the scope of use of Android's Uri implementation.  It's cool, but it's not very testable and java has one that's not great, but not terrifyingly difficult to work with.

Reply:
I am not sure if we want to use both Uri and URL... isnt just one of them sufficient for us?

The thing that adds to my confusion is that both these classes has their own definition of authority (hostname:port). Which we are using at places and it gets confused with our own definition of Authority (hostname/tenant)

So we have code like... authority.getAuthorityURL().getAuthority()